### DISCUSSION TITLE
Hardware revision descriptions

### DISCUSSION THREAD

**Discussion by ducky92** [sanni/cartreader#400] [General Chat]
Would it be possible to list the board revision changes some where? I have ordered the PCB's between HW5 rev 2 and the latest revision (rev 3 I believe) and I am not able to easily tell what was changed (and subsequently if I should order the new PCB before I start). I believe the change is related to this [discussion](https://github.com/sanni/cartreader/discussions/369) but reading over that I am not sure if I need the new board or not.

**Comment by ducky92** [comment]
re-reading that discussion post, that is not the one I was thinking of. https://github.com/sanni/cartreader/discussions/394 is where it was discussed to build hw5 rev3 of this project

**Comment by sanni** [comment]
There is no need to order new PCBs. All HW5 revisions are the same with the following minor differences:
- [Rev1](https://raw.githubusercontent.com/sanni/cartreader/d690968dc5700a3e6ef4d8498bd52778a7fff673/hardware/main_pcb_hw5/main_pcb_hw5.jpg) had the on/off 5V/3V switches on the left side
-  [Rev2](https://raw.githubusercontent.com/sanni/cartreader/66e4607a7f4440dbd290705ea275cc786db3019d/hardware/main_pcb_hw5/main_pcb_hw5.jpg) moved the on/off 5V/3V switches on the right side
-  [Rev3 (initial upload)](https://raw.githubusercontent.com/sanni/cartreader/cea70979eb036d1f41aaff85e3eadfbad5e72d9c/hardware/main_pcb_hw5/main_pcb_hw5.jpg) added a solder jumper on the back to disconnect the Arduino/Cart Slot voltage lines and 4 test points, both for troubleshooting purpose
-  [Rev3  (reupload)](https://raw.githubusercontent.com/sanni/cartreader/b3ac026495f3ee2cc23f5cdcbc9046549c9ccc64/hardware/main_pcb_hw5/main_pcb_hw5.jpg) has the exact same PCB layout as the previous Rev3 but has "TP1, TP2, TP3, TP4" written next to the testpoints.

**Reply by ducky92** [reply]
Thank you Sanni for the clarification.
